src/main/java/org/jenkinsci/test/acceptance/po/PluginManager.java

```diff
@@ -88,7 +88,7 @@ public void checkForUpdates() {
         // The check now button is a form submit (POST) with a redirect to the same page only if the check is successful.
         // We use the button itself to detect when the page has changed, which happens after the refresh has been done
         // And we check for the presence of the button again
-        WebElement checkButton = find(by.link("Check now"));
+        WebElement checkButton = find(by.css("form[action='checkUpdatesServer'] .jenkins-button"));
```

Comment:
better to use the id?

Reply:
The button prior to this PR didn't have an ID